test/cpp/naming/resolver_test_record_groups.yaml

```diff
@@ -137,11 +137,6 @@ resolver_component_tests:
     - {TTL: '2100', data: '2607:f8b0:400a:801::1002', type: AAAA}
     srv-ipv6-target-has-backend-and-balancer:
     - {TTL: '2100', data: '2607:f8b0:400a:801::1002', type: AAAA}
-
-resolver_component_tests_TODO:
-- 'TODO: enable this large-txt-record test once working. (it is much longer than 512
-  bytes, likely to cause use of TCP even if max payload for UDP is changed somehow,
-  e.g. via notes in RFC 2671)'
```

Comment:
nit: this looks like a pre-existing bug in this yaml file actually, but it looks like there are accidentally two
`record_to_resolve` key-value pairs in this "test group".

It looks like line 149, with `record_to_resolve: srv-ipv6-target-has-backend-and-balancer` is getting silently ignored. Can we please remove that one since the test is now getting used.

Reply:
Removed `record_to_resolve` in line 149. Thanks for the review!